3-enrich/stream-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.stream/good/StructEventWithContextSpec.scala

```diff
@@ -174,7 +175,7 @@ class StructEventWithContextSpec extends Specification {
       // "-1" prevents empty strings from being discarded from the end of the array
       val fields = enrichedEvent.toOption.get._1.split("\t", -1)
       fields.size must beEqualTo(StructEventWithContextSpec.expected.size)
-
+      enrichedEvent.toOption.get._3.get must_== StructEventWithContextSpec.pii
```

Comment:
can we add additional `isDefined` checks so that if it fails we'll have an idea why?

Reply:
👍